### PR TITLE
Adding grunt configuration for gesso

### PR DESCRIPTION
**Pull request by wwhurley**

### CODE REVIEW

File: web-starter/index.js [hunk 1]
```diff
@@ -48,6 +48,22 @@ module.exports = generators.Base.extend({
     });
   },
   configuring: {
+    addGruntTasks : function () {
```
Comment (gustavderdrache) [inline web-starter/index.js]: Style nit: colon spacing.

File: web-starter/index.js [hunk 2]
```diff
@@ -48,6 +48,22 @@ module.exports = generators.Base.extend({
...
+        shell.insertConfig('shell.gessoWatch', this.fs.read(this.templatePath('tasks/config/shell_gesso_watch.js')));
+        shell.insertConfig('shell.gessoBuild', this.fs.read(this.templatePath('tasks/config/shell_gesso_build.js')));
+        shell.loadNpmTasks('grunt-shell');
+        this.options.addDevDependency('grunt-shell', '^2.1.0');
+
+        this.options.getPlugin('grunt').registerWatchTask('shell:gessoWatch');
+        
+        this.options.getPlugin('grunt').registerTask('build', [{
+          task : 'shell:gessoBuild',
+          priority : 1
```
Comment (gustavderdrache) [inline web-starter/index.js]: Another colon spacing issue here, too.

**Review by gustavderdrache — CHANGES_REQUESTED** [review]
A few style nits, but looks good otherwise.

**Comment by wwhurley** [comment]
Used `eslint --fix` to resolve formatting issues.

**Review by gustavderdrache — APPROVED** [review]

**Pull request merged**